libs/pattern/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
This is a new file, we should use the latest license header as other files in this PR.

Reply:
Changed to Apache-2.0 only license header now.